docs/SERVER_CONFIGURATIONS.md

```diff
@@ -355,17 +360,17 @@ LHS_LISTENER_MY_LISTENER_CLIENT_ID_FILE=/temp/client_id
 
 ---
 
-### `LHS_LISTENER_<LISTENER NAME>_CLIENT_SECRET`
+### `LHS_OAUTH_CLIENT_SECRET`
```

Comment:
On one of our slack chats it was suggested that this should be named `LHS_OAUTH_SERVER_SECRET`. I didn't name it that way because `CLIENT_SECRET` and `CLIENT_ID` are well known oauth industry names, so I thought it would be better to use names that people already know. I can change it to `LHS_OAUTH_SERVER_SECRET` if you all don't like `LHS_OAUTH_CLIENT_SECRET`.